documentation/asciidoc/topics/attributes/product-attributes.adoc

```diff
@@ -0,0 +1,76 @@
+//
+// This file contains attributes for building Red Hat Data Grid documentation.
+//
+
+//
+// Attributes Shared with Community Docs
+//
+:brandname: Red Hat Data Grid
+:wildflybrandname: EAP
+:tomcatbrandname: JWS
+:infinispanversion: 9.4.0
+:moduleprefix: jdg
+:infinispanslot: jdg-7.3
+
+// Community Operator
+
+:ispn_operator: Data Grid Operator
+:k8s: OpenShift
+:okd: OpenShift Container Platform
+
+:operator_root: https://github.com/infinispan/infinispan-operator/
+:operator_raw: https://raw.githubusercontent.com/rigazilla/infinispan-operator/
```

Comment:
`infinispan` instead of `rigazzila`?

Reply:
I guess my previous comment should apply here because this file has product attributes. I don't think we want to use the community repo here. probably needs to be updated when the RHDG 7.3 productization work is complete.